torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -461,9 +472,8 @@ LLVMCodeGenImpl::LLVMCodeGenImpl(
       irb_(getContext()),
       kernel_func_name_(std::move(kernel_func_name)),
       bufsExtAlloc_(ExternalAllocBufFinder::find(stmt)) {
-#if LLVM_VERSION_MAJOR >= 15
-  context_->setOpaquePointers(false);
-#endif
+  // Enable opaque pointers feature by default.
+  context_->setOpaquePointers(true);
```

Comment:
opaque pointer feature is enabled in production after LLVM 15. I don't think you can always turn it on here.

Reply:
Thanks for feedback. Updated in the latest commit.